src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars stars--0">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line before the section element on line 16 to separate it from the closing body tag.